code/modules/mob/living/brain/brain_item.dm

```diff
@@ -151,13 +151,15 @@
 	if(user.zone_selected != BODY_ZONE_HEAD)
 		return ..()
 
-	if((C.head && (C.head.flags_cover & HEADCOVERSEYES)) || (C.wear_mask && (C.wear_mask.flags_cover & MASKCOVERSEYES)) || (C.glasses && (C.glasses.flags_1 & GLASSESCOVERSEYES)))
+	var/target_has_brain = C.getorgan(/obj/item/organ/brain)
+
+	if(!target_has_brain && C.is_eyes_covered())
```

Comment:
This will no longer give you the warning to remove their headwear if they already have a brain.

Reply:
So double brain?